livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/tts.py

```diff
@@ -135,13 +138,14 @@ def __init__(
             num_channels=AZURE_NUM_CHANNELS,
         )
 
+        speech_host = speech_host or os.environ.get("AZURE_SPEECH_HOST")
         speech_key = speech_key or os.environ.get("AZURE_SPEECH_KEY")
-        if not speech_key:
-            raise ValueError("AZURE_SPEECH_KEY must be set")
-
         speech_region = speech_region or os.environ.get("AZURE_SPEECH_REGION")
-        if not speech_region:
-            raise ValueError("AZURE_SPEECH_REGION must be set")
+
+        if not speech_host and (not speech_key or not speech_region):
```

Comment:
nit: slightly more readable
 
```suggestion
        if not speech_host and not (speech_key and speech_region):
```